frontend/src/components/Billing.js

```diff
@@ -24,4 +26,15 @@ class Billing extends Component {
     }
 };
 
-export default Billing;
+const mapStateToProps = function(state) {
```

Comment:
To keep it consistent with the rest of the groups work. Please convert this to es6 syntax